### PR TITLE
refactor: restructure entry point for web app

### PR DESCRIPTION
**Pull request by soofstad**
- Cleaner structure of the first few components of the web app
  - Group "CalculateFluidDialog" with the "CalculateFluid" component. (Main.tsx was getting bussy)
  - Separate Component for "Login checks" (Cleaner to keep out of page content)

### CODE REVIEW

File: api/src/entities/Multiflash.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from typing import Dict, List
+from typing import Dict, List  # isort: skip
 
-import libhg
```
Comment (mariuswinger) [inline api/src/entities/Multiflash.py:3]: Did you find out what the issue was?

Comment (soofstad) [reply api/src/entities/Multiflash.py:3]: no...

**Review by mariuswinger — APPROVED** [review]
Cleaner!

**Review by mariuswinger — APPROVED** [review]
Cleaner!

**Pull request merged**